packages/@tailwindcss-node/src/compile.ts

```diff
@@ -45,3 +46,21 @@ export async function compile(
     },
   })
 }
+
+// Attempts to import the module using the native `import()` function. If this
+// fails, it sets up `jiti` and attempts to import this way so that `.ts` files
+// can be resolved properly.
+let jiti: null | Jiti = null
+async function importModule(path: string): Promise<any> {
+  try {
+    return await import(path)
+  } catch (error) {
+    try {
+      if (!jiti) {
+        jiti = createJiti(import.meta.url, { moduleCache: false, fsCache: false })
+      }
```

Comment:
```suggestion
      jiti ??= createJiti(import.meta.url, { moduleCache: false, fsCache: false, interopDefault: true })
```

In v3 we use the `interopDefault` flag and I _think_ we need to here as well?

Reply:
I feel like it may have something to do with `module.exports =` or maybe `export =` in a TS file. Not 100% sure.

Reply:
Per documentation, this:

> Return the .default export of a module at the top-level.

Since we already so `then((m: any) => m.default ?? m)`, I think this is not necessary?

Reply:
Ah I think this helps interop with ESM that was compiled to CJS then loaded so yeah maybe we don't need the flag at all then.